docs/source/getting-started/first-feature.rst

```diff
@@ -78,11 +78,6 @@ Open a new terminal and run the following commands::
 
 The result is an up-to-date TypeScript interface to our DAML model, in particular to the new ``Message`` template and ``SendMessage`` choice.
```

Comment:
While we're at it, shouldn't it be like this?
```suggestion
The result is an up-to-date JavaScript interface to our DAML model, in particular to the new ``Message`` template and ``SendMessage`` choice.
```

Reply:
Not in this fix.